elm-tooling.json

```diff
@@ -0,0 +1,8 @@
+{
+    "tools": {
+        "elm": "0.19.1",
+        "elm-format": "0.8.5",
+        "elm-json": "0.2.13",
+        "elm-test-rs": "3.0.0"
```

Comment:
Before, elm-format and elm-json were not installed on docker since they are not required to run the analyzer.
I think it's not a huge deal, but if it is, can someone think of a good way to avoid that?

Reply:
alright, so trading consistency (using the same tools as in elm-tooling specification) for the weight of some(?) more dependency. I believe both `elm-json` and `elm-format` were pulled in via elm-review anyway right?

https://github.com/jfmengels/elm-review/blob/master/package.json#L15

I'm not super fluent with how the post-install of dependencies work, but if elm-review post-install was called, it certainly was pulling elm-format and elm-json. Makes sense since elm-review need both of them for some of its work.

Reply:
I didn't think about that, you are right, `elm-review` does pull these in, I double checked.

Reply:
Actually, sorry I was looking at the `elm-review` repo while the one installed via npm is [`node-elm-review`](https://github.com/jfmengels/node-elm-review). This one does not install elm-format, but does install elm-json for now, but it's not straightforward when looking at their package.json. In fact they are installing elm-json at runtime via a special function of elm-tooling package called `getExecutable`. Simon gave me more info about this:

> elm-review depends on elm-json ^0.2.10: https://github.com/jfmengels/node-elm-review/blob/2e1f46d0aa498641b9c48971c4edffc392f1dccb/lib/template-dependencies.js#L170
>
> And elm-tooling ^1.6.0: https://github.com/jfmengels/node-elm-review/blob/89f0546ce72e7bf6af22693ab02db3731ab5f160/package.json#L76
>
> What you want to do is to also depend on elm-tooling, on a version that is compatible with ^1.6.0. Then make sure that you only have one elm-tooling in your package-lock.json. (npm deduplicates things by default, but it’s possible to end up with duplicate dependencies still.)
>
> Then, add elm-json to your elm-tooling.json, the latest version that is compatible with ^0.2.10.
>
> This means that elm-review is guaranteed to use an elm-json version already installed by you.

Reply:
That doesn't change the fact that we are installing elm-format but I'm ok with the tradeoffs